liquibase-core/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -174,13 +174,13 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .build();
 
         HEADLESS = builder.define("headless", Boolean.class)
-                .setDescription("Force liquibase to think it has no access to a keyboard")
+                .setDescription("Force Liquibase to think it has no access to a keyboard")
                 .setDefaultValue(false)
                 .setCommonlyUsed(true)
                 .build();
 
         STRICT = builder.define("strict", Boolean.class)
-                .setDescription("Be stricter on allowed Liquibase configuration and setup?")
+                .setDescription("If true, restricts Liquibase configuration and setup: requires changeset author to be non-empty and requires LIQUIBASE_* environment variable names to be valid.")
```

Comment:
The strict setting also requires settings in the defaultsFile to be valid as well. I wonder if even now it's a lot to list the specifics of what strict mode does in this help description and instead we should make a docs page and point to that? At some point there will be enough strict details we'll have to, so maybe now?

I'd suggest giving a more "why" answer than the details of what it specifically does. Like: `If true, Liquibase enforces certain best practices and proactively looks for common errors. For more information see https://docs.liquibase.com/strict` and then have that strict page give details and examples about the properties file, env variables, the author tag, and whatever else we add down the road

Reply:
@nvoxland Your description works for me! I'm (slowly) creating individual pages for all the global arguments to provide more detailed usage/behavior descriptions and I can definitely make one for `strict` soon.

Though this begs the question of why we have so many disparate behaviors going through one argument. For example, if a user wants to be strict about envar names, but doesn't want to be strict on empty `author` values, shouldn't we support that via multiple distinct arguments? (Even if it's not a best practice, it's already not really a best practice to disable this in the first place.)